enactjs/sandstone
# WRP-6257: Layout: Added screenshot tests for `grow` prop of `Cell`

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
A follow-up PR to add qa-sampler and screenshot tests for `grow` prop in `Cell` (enactjs/enact#3128)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- ~~Added qa-sampler for `Layout`~~
- Added screenshot tests

~~qa-sampler~~
~~- One Column and Nested three Rows with a total of 9 Cells to control them at once.~~
~~- Three Cells in a Layout to control them individually.~~

~~In principle, this should be added to enact, but since enact does not have qa-sampler, it is added to sandstone instead.~~

screenshot tests

* Row
  - Simply set the `grow` prop to check if each cell occupies the same area.
  - Set `grow` and `size={1000} to make sure it fills the remaining area of the container.
  - by setting `grow` and `size={1500}`
     - Check that it does not shrink below the specified size
     - When `wrap` is activated, cells are listed in two lines and check if the last cell fills up the container.
    
* Column
  - Simply set the `grow` prop to check if each cell occupies the same area.
  - Set `grow` and `size={100}` to make sure it fills the remaining area of the container.
  - Set `grow` and `size={500}` to make sure it doesn't shrink below the
      specified size.

* Layout
  - by setting `grow` and `size={1500}`
     - Check that it does not shrink below the specified size.
     - When `wrap` is activated, check if cells are listed in two lines, and the last cell fills up the container.
 
### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
enactjs/enact#3128 should be merged first.

### Links
[//]: # (Related issues, references)
WRP-6257
enactjs/enact#3128

### Comments
Enact-DCO-1.0-Signed-off-by: Hoeun Ryu <hoeun.ryu@lge.com>
